src/Transactions/Builder/AbstractTransactionBuilder.php

```diff
@@ -20,20 +17,13 @@ public function __construct(?array $data = null)
         $this->transaction = $this->getTransactionInstance();
 
         $this->transaction->data = $data ?? [
-            'type'            => Types::EVM_CALL->value,
-            'typeGroup'       => TypeGroup::CORE,
-            'amount'          => '0',
-            'senderPublicKey' => '',
-            'fee'             => Fee::get(Types::EVM_CALL->value),
-            'nonce'           => '1',
-            'version'         => 1,
-            'network'         => Network::get()->pubKeyHash(),
-            'asset'           => [
-                'evmCall' => [
-                    'gasLimit' => 1000000,  // Default gas limit
-                    'payload'  => '',       // EVM code in hex format
-                ],
-            ],
+            'value'             => '0',
+            'senderPublicKey'   => '',
+            'gasPrice'          => '5',
+            'nonce'             => '1',
+            'network'           => Network::get()->pubKeyHash(),
+            'gasLimit'          => 1_000_000,
```

Comment:
can we set default gas limits for the supported transaction types? e.g.:
- transfer: 21000
- validator registration: 500000
- validator resignation: 150000
- vote: 200000
- unvote: 200000
- default: 1000000

I didn't see them being overwritten in the code in the builders, but maybe I overlooked something

Reply:
@ItsANameToo Deleted all the code related to fees, it's not actually being used anywhere